aea/aea.py

```diff
@@ -187,12 +181,15 @@ def _handle(self, envelope: Envelope) -> None:
         logger.debug("Handling envelope: {}".format(envelope))
         protocol = self.resources.get_protocol(envelope.protocol_id)
 
-        # TODO make this working for different skill/protocol versions.
+        # TODO specify error handler in config and make this work for different skill/protocol versions.
         error_handler = self.resources.get_handler(
-            DefaultMessage.protocol_id, ERROR_SKILL_ID,
+            DefaultMessage.protocol_id, DEFAULT_SKILL
         )
 
-        assert error_handler is not None, "ErrorHandler not initialized"
+        if error_handler is None:
+            logger.warning("ErrorHandler not initialized. Stopping AEA!")
```

Comment:
Throwing an assertion error here is messy, better to stop the AEA gracefully

Reply:
ok

Reply:
maybe do a `try except` on the whole `_handle` method?

Reply:
Why would we do that? Seems a bit brute force.